azure-quantum/tests/unit/test_job.py

```diff
@@ -127,13 +127,15 @@ def test_job_submit_microsoft_population_annealing(self):
         solver_type = functools.partial(microsoft.PopulationAnnealing, sweeps=200)
         solver_name = "PopulationAnnealing"
         self._test_job_submit(solver_name, solver_type)
-        self._test_job_submit(solver_name, solver_type, test_grouped=True)
+        # todo - re-enable once grouped model is live
+        #self._test_job_submit(solver_name, solver_type, test_grouped=True)
```

Comment:
I think we can leave this test live but instead just set `test_grouped` to `False`.
```suggestion
        self._test_job_submit(solver_name, solver_type, test_grouped=False)
```

Reply:
see comment below - I would prefer to leave it commented out so we don't run the same test twice